client/css/style.css

```diff
@@ -1401,12 +1401,11 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 #settings .opt {
 	display: block;
-	padding: 5px 0 10px 1px;
+	padding: 5px 0 5px 1px;
 }
 
 #settings .opt input {
-	float: left;
-	margin: 4px 10px 0 0;
+	margin-right: 6px;
```

Comment:
That float was just unnecessary, just affected the right margin, so this is roughly equivalent.